www/src/components/CodeBlock.tsx

```diff
@@ -23,30 +23,40 @@ import * as ParagonIcons from '~paragon-icons';
 import MiyazakiCard from './exampleComponents/MiyazakiCard';
 import HipsterIpsum from './exampleComponents/HipsterIpsum';
 import ExamplePropsForm from './exampleComponents/ExamplePropsForm';
+import { ContentCopy } from '../../../icons';
```

Comment:
nit: we have `~paragon-icons` alias for such imports

Reply:
done